src/ripple/app/misc/Transaction.h

```diff
@@ -171,6 +172,21 @@ class Transaction
     beast::Journal  j_;
 };
 
+struct TransactionEnvelope
+{
+public:
+    TransactionEnvelope (std::shared_ptr<Transaction> txn,
+                         std::string const& completeLedgers)
+        : txn (txn)
+        , completeLedgers (completeLedgers)
```

Comment:
We're taking the txn by value, we may as well move it (i.e. `tnx(std::move(txn))` I'd do something similar for `completeLedgers` as well, take the param by value and move it.

Nit: I'd rename the constructor params so they don't match the member vars.

Reply:
Removed TransactionEnvelope.